dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryImportData.java

```diff
@@ -0,0 +1,48 @@
+package com.dotcms.rest.api.v1.categories;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import org.glassfish.jersey.media.multipart.FormDataContentDisposition;
+import org.glassfish.jersey.media.multipart.FormDataParam;
+import java.io.InputStream;
+
+/**
+ * Runtime binding bean for multipart form parts.
+ */
+@Schema(name = "CategoryImportFormSchema", description = "Category import form data with CSV file and processing parameters")
+public class CategoryImportData {
+
+    private InputStream fileInputStream;
+    private FormDataContentDisposition fileDetail;
+    private String filter;
+    private String exportType;
+    private String contextInode;
+
+    @FormDataParam("file")
+    @JsonProperty("file")
+    @Schema(name = "file", description = "CSV file containing categories to import", type = "string", format = "binary")
```

Comment:
Have you been able to load this into the API Playground? 
Once, I attempted to document an InputStream sette,r and it broke the playground